src/defaults.mligo

```diff
@@ -3,29 +3,45 @@
 
 #include "types.mligo"
 
-let default_config : config = {
+let default_config (max_quorum, min_quorum, max_voting_period, min_voting_period : nat * nat * nat * nat): config = {
     proposal_check = (fun (params, extras : propose_params * contract_extra) -> true);
     rejected_proposal_return_value = (fun (proposal, extras : proposal * contract_extra) -> 0n);
     decision_lambda = (fun (proposal, extras : proposal * contract_extra) -> (([] : (operation list)), extras));
 
     max_proposals = 500n;
     max_votes = 1000n;
-    max_quorum_threshold = 1000n;
-    min_quorum_threshold = 1n;
-    max_voting_period = 2592000n; // 60 * 60 * 24 * 30
-    min_voting_period = 1n;
+    max_quorum_threshold = max_quorum;
+    min_quorum_threshold = min_quorum;
+    max_voting_period = max_voting_period;
+    min_voting_period = min_voting_period;
+    
+    //max_quorum_threshold = 1000n;
+    // min_quorum_threshold = 1n;
+    // max_voting_period = 2592000n; // 60 * 60 * 24 * 30
+    // min_voting_period = 1n;
     custom_entrypoints = (Map.empty : custom_entrypoints);
     }
 
-let default_storage (admin , token_address , now_val, metadata : address * address * timestamp * metadata_map) : storage = {
-    ledger = (Big_map.empty : ledger);
+let max (a, b : nat * nat) : nat =
+  if a > b
+  then a
+  else b
+
+let min (a, b : nat * nat) : nat =
+  if a < b
+  then a
+  else b
+
+
+let default_storage (admin, token_address, now_val, metadata, ledger, max_quorum, min_quorum, quorum_threshold, max_period, min_period, voting_period : address * address * timestamp * metadata_map * ledger * nat * nat * nat * nat * nat * nat) : storage = {
```

Comment:
One more suggestion about this is that we now have a bit too many parameter to this function (and the corresponding one for `registryDAO` and `treasuryDAO`).

So I suggest to create a new type just to carry this information, like:
```ocaml
type initial_data =
  { admin : address
  ; token_address : address
  ; now_val : timestamp
  <...>
```

In the `Makefile` we could then use it as:
```
	$(BUILD_STORAGE) --output-file $(OUT)/trivialDAO_storage.tz src/base_DAO.mligo base_DAO_contract 'default_full_storage({admin = "$(admin_address)"; token_address = "$(token_address)"; now_val = $(now_val); <...> })'
```